ipywidgets/widgets/widget_float.py

```diff
@@ -159,6 +159,8 @@ class FloatSlider(_BoundedFloat):
         default is 'horizontal', orientation of the slider
     readout : {True, False}
         default is True, display the current value of the slider next to it
+    behaviour : str
```

Comment:
Commenting here as this came up during the weekly meeting today, with @ianhi spotting this attribute was not in American English:

```suggestion
    behavior : str
```

Reply:
Ah that was probably because of `noUISlider` using `behaviour` in the options:

https://refreshless.com/nouislider/slider-options/#section-limit

![image](https://user-images.githubusercontent.com/591645/133807331-4f0be361-8526-4097-86d0-227b5f705c7c.png)